lib/rails_charts/line_chart.rb

```diff
@@ -68,11 +75,23 @@ def defaults
         },
         toolbox: {
           feature: {
-            saveAsImage: {}
+            saveAsImage: {show: false}
```

Comment:
is this the same issue as my stack: {} for stacked_bar_chart? like passing the option has no effect?

Reply:
yeah it is. For now, I think we probably don't want this option so I'm opted to keep this as default.